common/src/main/java/bisq/common/crypto/KeyStorage.java

```diff
@@ -58,6 +68,13 @@
 @Singleton
 public class KeyStorage {
     private static final Logger log = LoggerFactory.getLogger(KeyStorage.class);
+    private static final int SALT_LENGTH = 20;
+    private static final int SECRET_ITERATIONS = 65536;
```

Comment:
how long does it take on your computer with this number of iterations?

Reply:
I added timing to the key derivation and on my computer it takes about 200ms. The Scrypt derivation takes about 1 second.

Reply:
Deleted